dataCollection/settings.py.template

```diff
@@ -207,39 +201,11 @@ LOGGING = {
         },
         'django.request': {
             'handlers': ['request_handler'],
-            #'level': 'ERROR',
              'level': 'DEBUG',
-            #'propagate': True
              'propagate': False
-        },
-        #'django': {
-        #    'handlers': ['logfile'],
-        #    'level': 'ERROR',
-        #    'propagate': False,
-        #}
+        }
     }
 }
-#LOGGING = {
-#    'version': 1,
-#    'disable_existing_loggers': False,
-#    'filters': {
-#        'require_debug_false': {
-#            '()': 'django.utils.log.RequireDebugFalse'
-#        }
-#    },
-#    'handlers': {
-#        'mail_admins': {
-#            'level': 'ERROR',
-#            'filters': ['require_debug_false'],
-#            'class': 'django.utils.log.AdminEmailHandler'
-#        },
-#        'console': {
-#            'level': 'DEBUG',
-#            'class': 'logging.StreamHandler',
-#        },
-#    }
-#}
-
 
 KEY = '52af783df367b7a5c7d01457d82544bf'
 SALT = '58268845878249858254353589767595'
```

Comment:
I have a bad feeling about that API key. That goes with Google's cloud messaging service. We need to check if devices are trying to send data back to the server through that (I don't remember if we confirmed they talk directly to the server) because if they are, we may need to get a new API key and make sure the API calls are happening correctly as that could be why location reporting isn't working.